Both packages (2.12 and 2.13) build but won't execute. Might be a shell quoting problem?
```result/bin/amm
Error: Could not find or load main class ammonite.Main --no-remote-logging
Caused by: java.lang.ClassNotFoundException: ammonite.Main --no-remote-logging
```

--- a/pkgs/development/tools/ammonite/default.nix
+++ b/pkgs/development/tools/ammonite/default.nix
@@ -23,7 +23,7 @@ let
         install -Dm755 $src $out/bin/amm
         sed -i '0,/java/{s|java|${jre}/bin/java|}' $out/bin/amm
       '' + optionalString (disableRemoteLogging) ''
-        sed -i '0,/ammonite.Main/{s|ammonite.Main|ammonite.Main --no-remote-logging|}' $out/bin/amm
+        sed -i "0,/ammonite.Main/{s|ammonite.Main'|ammonite.Main' --no-remote-logging|}" $out/bin/amm
         sed -i '1i #!/bin/sh' $out/bin/amm
       '';
 

